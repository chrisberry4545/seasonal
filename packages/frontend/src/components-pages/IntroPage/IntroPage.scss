@import '../../styles/settings';
@import '../../styles/tools';

.c-intro-page {
  width: 100%;
  text-align: center;
  max-width: 1024px;
  margin: auto;
}

  .c-intro-page__top-section {
    border-bottom: 1px solid $grey-med;
    padding-bottom: 2.5rem;
  }

  .c-intro-page__splash-wrapper {
    padding-bottom: 48.828125%;
    position: relative;
  }

  .c-intro-page__splash-image {
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
  }

  .c-intro-page__mobile-extra-app-links {
    margin-top: 3rem;
    @include mq($from: tablet) {
      display: none;
    }
  }
