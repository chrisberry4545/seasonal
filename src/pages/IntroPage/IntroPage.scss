@import './../../styles/settings';
@import './../../styles/tools';

.c-intro-page {
  width: 100%;
  text-align: center;
  max-width: 1024px;
  margin: auto;
}

  .c-intro-page__top-section {
    border-bottom: 1px solid $grey-med;
    padding-bottom: 2.5rem;
  }

  .c-intro-page__splash-image {
    width: 100%;
  }

  .c-intro-page__mobile-extra-app-links {
    margin-top: 2rem;
    @include mq($from: tablet) {
      display: none;
    }
  }
