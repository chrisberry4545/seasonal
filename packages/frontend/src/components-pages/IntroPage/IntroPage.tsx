import React, { FC } from 'react';

import './IntroPage.scss';

import {
  AboutEatSeasonal,
  ViewAppLinks,
  WhyEatSeasonal
} from '../../components-main';

export const IntroPage: FC = () => (
  <div className='c-intro-page'>
    <div className='c-intro-page__top-section'>
      <img alt='Eat Seasonal splash'
        className='c-intro-page__splash-image' src='/splash.png'/>
      <ViewAppLinks />
    </div>
    <AboutEatSeasonal />
    <WhyEatSeasonal />
    <div className='c-intro-page__mobile-extra-app-links'>
      <ViewAppLinks />
    </div>
  </div>
);
