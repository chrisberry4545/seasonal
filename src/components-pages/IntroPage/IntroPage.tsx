import React from 'react';

import './IntroPage.scss';

import {
  ViewAppLinks,
  WhyEatSeasonal
} from '../../components-main';

export const IntroPage = () => (
  <div className='c-intro-page'>
    <div className='c-intro-page__top-section'>
      <img className='c-intro-page__splash-image' src='/splash.png'/>
      <ViewAppLinks />
    </div>
    <WhyEatSeasonal />
    <div className='c-intro-page__mobile-extra-app-links'>
      <ViewAppLinks />
    </div>
  </div>
);
