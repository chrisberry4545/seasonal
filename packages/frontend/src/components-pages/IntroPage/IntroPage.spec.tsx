import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { IntroPage } from '../IntroPage/IntroPage';

jest.mock('../../components-main', () => ({
  AboutEatSeasonal: () => 'AboutEatSeasonal',
  ViewAppLinks: () => 'ViewAppLinks',
  WhyEatSeasonal: () => 'WhyEatSeasonal'
}));

describe('<IntroPage />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <IntroPage />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
