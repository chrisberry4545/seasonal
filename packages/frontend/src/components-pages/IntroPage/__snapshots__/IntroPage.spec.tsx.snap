// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroPage /> renders correctly 1`] = `
<div
  className="c-intro-page"
>
  <div
    className="c-intro-page__top-section"
  >
    <div
      className="c-intro-page__splash-wrapper"
    >
      <img
        alt="Eat Seasonal splash"
        className="c-intro-page__splash-image"
        src="/splash.png"
      />
    </div>
    <ViewAppLinks />
  </div>
  <AboutEatSeasonal />
  <WhyEatSeasonal />
  <div
    className="c-intro-page__mobile-extra-app-links"
  >
    <ViewAppLinks />
  </div>
</div>
`;
